<?xml version="1.0"?>
<launch>
  <!-- Run MTC pick and place -->

  <rosparam command="load" file="$(find o2ac_routines)/config/spawn_object.yaml" ns="o2ac_spawn_object"/>

  <node name="spawn_object" pkg="o2ac_routines" type="spawn_object.py">
  </node>
  
</launch>
